newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -69,21 +79,27 @@ export default class SearchPanel extends PureComponent<Props, State> {
       replaceText,
       searchInSelection,
       matchCase,
+      searchInActions,
+      searchInConditions,
     } = this.state;
-    if (searchDirty) {
-      showMessageBox(
-        'Click on Search first, inspect the results and then click on Replace to do the replacement(s).'
-      );
-      return;
-    }
+    // if (searchDirty) {
+    //   showMessageBox(
+    //     'Click on Search first, inspect the results and then click on Replace to do the replacement(s).'
+    //   );
+    //   return;
+    // }
+
+    this.launchSearch();
 
     this.props.onReplaceInEvents({
       searchInSelection,
       searchText,
       replaceText,
       matchCase,
-      searchInActions: true,
-      searchInConditions: true,
+      searchInActions,
+      searchInConditions,
+      searchInActions,
```

Comment:
You've entered twice searchInActions and searchInConditions, you can remove the second occurence of each